lib/base64.c

```diff
@@ -283,13 +283,13 @@ static CURLcode base64_encode(const char *table64,
  *
  * Given a pointer to an input buffer and an input size, encode it and
  * return a pointer in *outptr to a newly allocated memory area holding
- * encoded data. Size of encoded data is returned in variable pointed by
- * outlen.
+ * encoded data. Size of the encoded data is returned in a variable pointed
+ * to by *outlen.
  *
- * Input length of 0 indicates input buffer holds a NUL-terminated string.
+ * Input length of 0 indicates input buffer holds a 0-terminated string.
```

Comment:
We call them "NUL-terminated" (or null) through-out the project since 032e838b73578e. I don't think changing the name of them in a few comments is a good idea.